docs/practices/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Good Practices
+weight: 800
+---
+# Adopt good practices for projects
+
+We strive to achieve a high degree of quality and maturity for open source projects hosted by the OpenRail Association. Our incubation process supports this by gradually setting requirements and recommendations for good practices to be adopted by projects. Requirements become stricter with advancing in the incubation process, so that the different incubation process transparently guarantee a certain level of maturity.
```

Comment:
```suggestion
We strive to achieve a high degree of quality and maturity for open source projects hosted by the OpenRail Association. Our incubation process supports this by gradually setting requirements and recommendations for good practices to be adopted by projects. Requirements become stricter with advancing in the incubation process, so that the different incubation processes transparently guarantee a certain level of maturity.
```

Reply:
Fixed it differently.